### ISSUE TITLE
Feature: Add fuzziness to integration tests

### ISSUE THREAD

**Issue by yardasol** [arfc/saltproc#78]
**Background and motivation**
For all the details, see the discussion in #76 as well as issues referenced there. 

TL;DR, the Serpent integration tests have an effectively non-public cross section library hardcoded into their input. As SaltProc is open source, we should make it so that anyone who downloads the software is able to run the integration tests (provided the have the relevant licenses for their depletion code software of choice. That isn't an issue with OpenMC thankfully, but we're still getting support for that set up)

**Description of idea**
We should have some fuzziness in our Serpent integration tests so that a user could use any cross section library and have the integration tests pass.

**Implementation details**
- In the Serpent integration test files, use `assert_almost_equal` statements instead of `assert_equal` statements.
- Decouple the Serpent integration test input files from a particular cross section library.

**Potential snags**
- We may not be able to completely decouple the Serpent integration test input files from a particular cross section library if there aren't default options for certain parameters:
